PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/PennCourseAlertPagerAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package com.pennapps.labs.pennmobile.adapters
+
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.pennapps.labs.pennmobile.PennCourseAlertCreateAlertFragment
+import com.pennapps.labs.pennmobile.PennCourseAlertManageAlertsFragment
+
+class PennCourseAlertPagerAdapter(fragment: Fragment): FragmentStateAdapter(fragment) {
+    override fun getItemCount(): Int {
+        return COUNT
+    }
+
+    override fun createFragment(position: Int): Fragment {
+        return if (position == CREATE_ALERT_POSITION) {
+            PennCourseAlertCreateAlertFragment()
+        } else {
+            PennCourseAlertManageAlertsFragment()
+        }
+    }
+
+    companion object {
+        private const val CREATE_ALERT_POSITION = 0
+//        private const val MANAGE_ALERTS_POSITION = 1
+        const val COUNT = 2
```

Comment:
what is the 0/1/2 mean here? Just for clarity

Reply:
The 0 and 1 are just indices for the tabs in ViewPager2. The 0 variable is used in the same file above to reference which tab to show based on the ViewPager2 variable `position`. The 1 variable is not used since there is only two cases (0 and 1) and I am using a return block with if/else above on lines 14-16. I left it commented for future reference in case we want to include more tabs we can just add them there and modify `createFragment()` accordingly. The use of the companion object is the preferred way to manage ViewPager2 adapter as opposed to the old method of ViewPager.

The COUNT variable is just the standard for ViewPager2 adapter to return in `getItemCount()`. It reflects the number of tabs.